src/Razor/Microsoft.AspNetCore.Razor.Language/src/SourceSpan.cs

```diff
@@ -91,33 +85,11 @@ public override string ToString()
 
         public static bool operator ==(SourceSpan left, SourceSpan right)
         {
-            if (ReferenceEquals(left, right))
-            {
-                // Exact equality e.g. both objects are null.
-                return true;
-            }
-
-            if (ReferenceEquals(left, null))
-            {
-                return false;
-            }
-
             return left.Equals(right);
```

Comment:
Note - safe because `SourceSpan` is a struct, so can't be null, and will never be `ReferenceEquals`.